src/renderers/shared/fiber/__tests__/ReactIncrementalPerf-test.js

```diff
@@ -382,7 +382,7 @@ describe('ReactDebugFiberPerf', () => {
     expect(getFlameChart()).toMatchSnapshot();
   });
 
-  it('deduplicates lifecycle names during commit to reduce overhead', () => {
+  xit('deduplicates lifecycle names during commit to reduce overhead', () => {
```

Comment:
Why do we need to disable this? I don't think it tests specifically progressed work.